egs/thchs30/s5/local/dict/extra_questions.txt

```diff
@@ -0,0 +1,7 @@
+sil
+a1 ai1 an1 ang1 ao1 e1 ei1 en1 eng1 i1 ia1 ian1 iang1 iao1 ie1 in1 ing1 iong1 iu1 ix1 iy1 o1 ong1 ou1 u1 ua1 uai1 uan1 uang1 ueng1 ui1 un1 uo1 v1 van1 ve1 vn1 
```

Comment:
The vocabulary-related things like the lexicon and extra_questions.txt and so on should be a download from openslr.org--- or better, just make the lexicon the download and make everything else derived from it.  If we added all the lexicons of the various recipes to the Kaldi repository it would get huge very fast.